package.json

```diff
@@ -56,13 +56,13 @@
   "scripts": {
     "docs": "jsdoc -c .jsdoc.js",
     "generate-scaffolding": "repo-tools generate all --except lib_readme",
-    "lint": "eslint src/ samples/ system-test/ test/",
-    "prettier": "prettier --write src/*.js src/*/*.js samples/*.js samples/*/*.js test/*.js test/*/*.js system-test/*.js system-test/*/*.js",
+    "lint": "eslint samples/",
```

Comment:
why are we dropping linting on certain dirs?

Reply:
The module is in TypeScript now :) `gts check` is handling most of the work for TypeScript.  The only place we need prettier and eslint now are samples.